apps/backend/src/reconciliation/pos-reconciliation.service.ts

```diff
@@ -26,7 +26,7 @@ export class POSReconciliationService {
       const payment = payments.find(
         (itm) =>
           itm.amount == parseFloat(deposit.transaction_amt.toString()) &&
-          itm.method == deposit.method
+          itm.method == deposit.card_vendor
```

Comment:
This will not match, as the method is an sbc_garms code, and the pos deposit table is using a join to set method to match this code. This will require updates elsewhere.

Reply:
(I know that as discussed we are planning to use the master data payment method table to match all payments/deposits, but at the time of writing this I was still under the assumption that we'd be handling all joins post insert to the db, and that reconciliation would be SBC specific for MVP). I can set the join on the payment table to match the card vendor if that is preferable?